app/views/users/team.html.haml

```diff
@@ -5,7 +5,7 @@
     .column-two-thirds
       %h1.heading-large= t('views.users.team.page_title')
     .column-one-third
-      = link_to t('views.users.new.page_title'), new_user_invitation_path, class: "button align-with-heading"
+      = link_to t('views.users.new.page_title'), new_user_invitation_path('advanced'), class: "button align-with-heading"
```

Comment:
I think there is a need for this param but actually in this view we don't need it since we'd want the user to select basic or advanced. When invited a custodian or admin user we know the role before hand so passing in the param saves the user time